concourse/pipelines/templates/anchors/destroy-cluster-tpl.yml

```
@@ -0,0 +1,26 @@
+{# Template for an anchor that destroys a cluster.
+ #
+ # Expects a 'cluster_type' variable to be set to either 'gpdb' or 'hadoop'
+ #}
+
+{% if cluster_type == 'gpdb' %}
+    {% set put_suffix           = 'gpdb' %}
+    {% set env_name_file_suffix = '-gpdb' %}
+    {% set terraform_source     = 'ccp_src/google/' %}
+{% else %}
+    {% set put_suffix           = 'ipa-hadoop' %}
+    {% set env_name_file_suffix = '-ipa-hadoop' %}
+    {% set terraform_source     = 'pxf_src/concourse/terraform/ipa-multinode-hadoop' %}
+{% endif %}
+
+- &destroy_[[cluster_type]]_cluster
+  put: terraform-[[put_suffix]]
+{% if cluster_type == 'gpdb' %}
+  resource: terraform
+{% endif %}
+  params:
+    action: destroy
+    env_name_file: terraform[[env_name_file_suffix]]/name
```

```suggestion
    env_name_file: terraform-[[put_suffix]]/name
```

something like that, maybe a better variable name.